date_range/tests/models.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class TestDateRangeSearchMixin(models.Model):
```

Comment:
```suggestion
class TestDateRangeSearchMixin(models.TransientModel):
    _auto = False
```

Reply:
hmmno, the `_auto` defeats the purpose, but TransientModel is helpful

Reply:
Fixed it by cleaning out _inherit_children even more aggressively.